azurerm/internal/services/network/tests/application_gateway_resource_test.go

```diff
@@ -4011,6 +4040,207 @@ resource "azurerm_application_gateway" "test" {
 `, template, data.RandomInteger)
 }
 
+func testCheckAzureRMApplicationGatewayChangeCert(resourceName, certName string) resource.TestCheckFunc {
+	return func(s *terraform.State) error {
+		client := acceptance.AzureProvider.Meta().(*clients.Client).Network.ApplicationGatewaysClient
+		ctx := acceptance.AzureProvider.Meta().(*clients.Client).StopContext
+
+		// Ensure we have enough information in state to look up in API
+		rs, ok := s.RootModule().Resources[resourceName]
+		if !ok {
+			return fmt.Errorf("Not found: %s", resourceName)
+		}
+
+		gatewayName := rs.Primary.Attributes["name"]
+		resourceGroup := rs.Primary.Attributes["resource_group_name"]
+
+		agw, err := client.Get(ctx, resourceGroup, gatewayName)
+		if err != nil {
+			return fmt.Errorf("Bad: Get on ApplicationGatewaysClient: %+v", err)
+		}
+
+		newSslCertificates := make([]network.ApplicationGatewaySslCertificate, 1)
+		newSslCertificates[0] = network.ApplicationGatewaySslCertificate{
+			Name: &certName,
+			Etag: (*agw.SslCertificates)[0].Etag,
+			Type: (*agw.SslCertificates)[0].Type,
+			ID:   (*agw.SslCertificates)[0].ID,
+		}
+
+		agw.SslCertificates = &newSslCertificates
+
+		future, err := client.CreateOrUpdate(ctx, resourceGroup, gatewayName, agw)
+		if err != nil {
+			return fmt.Errorf("Bad: updating AGW: %+v", err)
+		}
+
+		if err := future.WaitForCompletionRef(ctx, client.Client); err != nil {
+			return fmt.Errorf("Bad: waiting for update of AGW: %+v", err)
+		}
+
+		return nil
+
```

Comment:
remove this newline

Reply:
why?

Reply:
I believe this is what is upsetting the linter.